verilog/formatting/token_annotator.cc

```diff
@@ -426,6 +431,11 @@ static WithReason<int> SpacesRequiredBetween(
 
     // Spacing in ranges
     if (InRangeLikeContext(right_context)) {
+      // Bit selection and indexing inside []
+      if (style.compact_non_declaration_ranges && !InDeclaredDimensions(right_context)) {
+        return {0, "Limit spaces before ':' in bit slice/indexing to 0"};
+      }
```

Comment:
I suggest we leave this out this time (changing spacing around `:`), and only target the binary operators as you've done above.

Reply:
Removed.